supervisor_types.go

```diff
@@ -127,7 +129,12 @@ type SpatialDimension struct {
 type TransformSet []Transform
 
 // DimensionSet is a unique set of druid datasource dimensions(labels).
-type DimensionSet []string
+type DimensionSet []interface{}
```

Comment:
is there any reason why it's not defined as `type DimensionSet []Dimension`?

Reply:
nasty stuff, I've seen something similar with `queryGranularity` response where there can be a string or object in scope of one field, depending on value